dev-tools/startup/eth-ganache.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+npx ganache \
+  --server.host 0.0.0.0 \
+  --wallet.deterministic \
+  --wallet.totalAccounts 50 \
+  --database.dbPath /usr/db \
+  --chain.networkId 12345 &
```

Comment:
this chainId is static and never changes across runs, so we're just hardcoding a random id?